### PR TITLE
Add Many Worlds token (BEP20)

### PR DESCRIPTION
**Pull request by gazjones00**
As requested by @Cryptocool1, an updated PR for the Many Worlds token

**• What is the purpose of this token? How will it benefit Trust Wallet users?**

$MANY is an innovative DeFi token built on the Binance Smart Chain network. Not only is it an incredible investment asset on its own, but it is also intended to serve as the governing token for blockchain solutions in various stages of development.

At the heart of the smart contract is a sophisticated set of tokenomics designed to overcome fundamental flaws in traditional rewards based tokens. Built into the smart contract structure is a token ageing and leveling system which makes these tokenomics possible; this is a novel concept never seen before in the blockchain space. 

The team, committed to transparency, accountability, perseverance, and ingenuity, will play a vital role in the long term viability of this project.

**• How is the token being distributed? Are you conducting an airdrop?**
There were some initial airdrops to our Private sale investors, we also used Pinksale to distrbute funds. Now the token is being distributed via the PancakeSwap DEX

**• Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?**
We did a presale here - https://www.pinksale.finance/#/launchpad/0x40aA8469b2AEFD263d2604DB38C5474b3BC7d3A3?chain=BSC

**• Is the token listed on a centralized/decentralized exchange?**
Via the PancakeSwap DEX, we're in talks with other centralized exchanges currently

**• Is a Smart Contract audit completed? Where can we find this information?**
We are double audited, you can see both below;

https://solidity.finance/audits/ManyWorlds/
https://github.com/interfinetwork/smart-contract-audits/blob/audit-updates/ManyWorlds_AuditReport_InterFi.pdf

### CODE REVIEW

File: blockchains/smartchain/assets/0x3E2ed75F718251Fd011312b9E33Bf10A4dbfda54/info.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            "name": "reddit",
+            "url": "https://reddit.com/r/ManyWorldsToken/"
+        },
+        {
+            "name": "coinmarketcap",
+            "url": "https://coinmarketcap.com/currencies/many-worlds-token/"
+        },
+        {
+            "name": "coingecko",
+            "url": "https://www.coingecko.com/en/coins/many-worlds"
```
Comment (cryptomanz) [inline blockchains/smartchain/assets/0x3E2ed75F718251Fd011312b9E33Bf10A4dbfda54/info.json:30]: this is correct url: "https://coingecko.com/coins/many-worlds", please change

File: blockchains/smartchain/assets/0x3E2ed75F718251Fd011312b9E33Bf10A4dbfda54/info.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            "name": "github",
+            "url": "https://github.com/Many-Worlds-Token/"
+        },
+        {
+            "name": "twitter",
+            "url": "https://twitter.com/manyworldstoken"
+        },
+        {
+            "name": "reddit",
+            "url": "https://reddit.com/r/ManyWorldsToken/"
```
Comment (cryptomanz) [inline blockchains/smartchain/assets/0x3E2ed75F718251Fd011312b9E33Bf10A4dbfda54/info.json:22]: this is correct url: "https://reddit.com/ManyWorldsToken/", please change

**Comment by gaz-think3** [comment]
> Hello, Please follow the rules for adding token, your logo should be a full screen, 256x256.

Hey @cryptomanz, just confirming you need this to be square rather than circular

**Comment by Cryptocool1** [comment]
it will be circle, but the size should be 256/256

**Comment by gaz-think3** [comment]
> it will be circle, but the size should be 256/256

Understood, I think this has been resolved - thank you

**Comment by Cryptocool1** [comment]
your size is W: 512px | H: 512px, better to do W: 256px | H: 256px

File: blockchains/smartchain/assets/0x3E2ed75F718251Fd011312b9E33Bf10A4dbfda54/info.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        },
+        {
+            "name": "coinmarketcap",
+            "url": "https://coinmarketcap.com/currencies/many-worlds-token/"
+        },
+        {
+            "name": "coingecko",
+            "url": "https://www.coingecko.com/coins/many-worlds"
```
Comment (cryptomanz) [inline blockchains/smartchain/assets/0x3E2ed75F718251Fd011312b9E33Bf10A4dbfda54/info.json:30]: please remove ```www```

File: blockchains/smartchain/assets/0x3E2ed75F718251Fd011312b9E33Bf10A4dbfda54/info.json (hunk 2 above)
Comment (cryptomanz) [inline blockchains/smartchain/assets/0x3E2ed75F718251Fd011312b9E33Bf10A4dbfda54/info.json:22]: https://reddit.com/ManyWorldsToken/ its correct url, please change

Comment (gaz-think3) [reply blockchains/smartchain/assets/0x3E2ed75F718251Fd011312b9E33Bf10A4dbfda54/info.json:22]: Hey @cryptomanz, can you verify this I get a 404 with that URL

Screenshot 2022-01-26 at 18 20 24

**Comment by gaz-think3** [comment]
Just confirming that has been paid transaction hash here:
 https://explorer.binance.org/tx/89AF7F5900AF0D37ADC3F743B25092141F40C1A6D37389DB80C0252D1A4CD19D

Thanks

**Comment by cryptomanz** [comment]
As per our policy, brand new tokens are not included, there is a minimum requirement of 5,000 holder addresses and 10,000 transactions, in order to filter out scam tokens.
See the criteria at : https://developer.trustwallet.com/assets/requirements .
Please come back to us when all criteria are met.
When creating a new PR make sure to mention this PR, so that we can see that fee has been paid already here.

**Comment by cryptomanz** [comment]
As per our policy, brand new tokens are not included, there is a minimum requirement of 5,000 holder addresses and 10,000 transactions, in order to filter out scam tokens.
See the criteria at : https://developer.trustwallet.com/assets/requirements . On hold for now.

**Comment by gaz-think3** [comment]
> As per our policy, brand new tokens are not included, there is a minimum requirement of 5,000 holder addresses and 10,000 transactions, in order to filter out scam tokens. See the criteria at : https://developer.trustwallet.com/assets/requirements . On hold for now.

This was discussed prior, we've had discuss with TW Support and we were pointed to TW Dev Telegram. Where I picked up the conversation with @Cryptocool1.

**Comment by gaz-think3** [comment]
> How will your token benefit Trust Wallet users?

$MANY is a token that day-by-day is increasing its holders. We've partnered with major players in the Finance world, i.e. Cloud Payments, to deliver a ramp-like payment solution governed by the $MANY token. Trust wallet will provide convenience, integrity and confidence to our investors and community by making $MANY tokens accessible via our preferred wallet of choice. We're building a unique onboarding experience in which we recommend using Trust wallet. Therefore, ensuring that we are directing more users towards Trust wallet as we grow.

Furthermore, $MANY have a built-in dividend distribution system that rewards users who hold the token with BUSD (BEP20). Trust Wallet will make it more convenient for investors to see their accumulated rewards.

We would also like to mention that we would be happy to set up an online video call to discuss further if this would be useful. Our interoperability utility is still under NDA, so we can't disclose all of this on this public thread. But, we'd be happy to discuss this in private.

Kind regards,
Gaz

**Comment by cryptomanz** [comment]
Please come back to us when all criteria are met.
When creating a new PR make sure to mention this PR, so that we can see that fee has been paid already here.

**Pull request closed without merging**